adapter/pkg/logging/log_format.go

```diff
@@ -0,0 +1,117 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package logging
+
+import (
+	"bytes"
+	"fmt"
+	"runtime"
+	"strings"
+
+	logrus "github.com/sirupsen/logrus"
+	"github.com/wso2/product-microgateway/adapter/pkg/config"
+)
+
+type plainFormatter struct {
+	TimestampFormat string
+	LevelDesc       []string
+}
+
+// ErrorHook for adding a custom logic to default the error details only for error logs
+type ErrorHook struct {
+}
+
+// LogFormatter is to keep the relevant log formatter
+var (
+	LogFormatter logrus.Formatter
+)
+
+func init() {
+	InitLogFormatter()
```

Comment:
If there won't be any line additions to this init() method, isn't it better to use the InitLogFormatter() directly?

Reply:
used the init() method itself to put the logic was in InitLogFormatter.
fixed in https://github.com/wso2/product-microgateway/pull/2633/commits/c6a5a466f9b3caaeca46c23fda940a37e091dd7c